packages/react-native-win32/src/Libraries/Components/Text/Tests/TextWin32Test.tsx

```diff
@@ -0,0 +1,164 @@
+'use strict';
+import * as React from 'react';
+import { Button, View } from 'react-native';
+import { TextWin32 } from '../TextWin32';
+import { ViewWin32 } from '../../View/ViewWin32';
+
+// Disabling no-jsx-lambda so functional components are more convenient to use
+
+const TextRunsTest: React.FC<{}> = () => {
+  return (
+    <View focusable>
+      <TextWin32>
+        <TextWin32>Text With </TextWin32>
+        <TextWin32>Multiple Text </TextWin32>
+        <TextWin32>Children [Runs]</TextWin32>
+      </TextWin32>
+    </View>
+  );
+};
+
+const FocusableTextTest: React.FC<{}> = () => {
+  return (
+    <ViewWin32>
+      <TextWin32 selectable>This TextWin32 demonstrates focusable</TextWin32>
```

Comment:
should this be focusable?
```suggestion
      <TextWin32 focusable>This TextWin32 demonstrates focusable</TextWin32>
```

Reply:
sure thought I'd fixed that.  Maybe just in the other PR.